Makefile

```diff
@@ -43,16 +46,17 @@ help:
 	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
 	@echo "  coverage   to run coverage check of the documentation (if enabled)"
 	@echo "  dummy      to check syntax errors of document sources"
+	@echo "  images     to make drawio images"
```

Comment:
Remove this line, there is no corresponding target to make the images, and I'm not sure we want that at this stage.